drivers/perf/resctrl_pmu.c

```diff
@@ -144,7 +144,7 @@ static int resctrl_pmu_event_init(struct perf_event *event)
 			return -EINVAL;
 	}
 
-	hwc->idx = resctrl_arch_mon_ctx_alloc_no_wait(r, event_num);
+	hwc->idx = resctrl_arch_mon_ctx_alloc(r, event_num);
```

Comment:
I can not find this line on the commit efe7525 on MPAM-dev.

Reply:
this is intergrated to resctrl_arch_mon_ctx_alloc,
the resctrl_arch_mon_ctx_alloc is a no_wait version.